sdk/core/core-client-rest/src/common.ts

```diff
@@ -43,12 +43,6 @@ export type RequestParameters = {
   allowInsecureConnection?: boolean;
   /** Set to true if you want to skip encoding the path parameters */
   skipUrlEncoding?: boolean;
-  /**
-   * With this property set to true, the response body will be returned
-   * as a binary array UInt8Array
-   */
-  binaryResponse?: boolean;
```

Comment:
looks like we are removing binary related reponse handling, I wonder if it's because we can have Streamable now ?

Reply:
Yeah, this was added before Streaming was possible. Now it doesn't add much value as users streaming responses will get it as byte array.